framework/core/src/Discussion/Search/FulltextFilter.php

```diff
@@ -84,9 +103,69 @@ public function search(SearchState $state, string $value): void
             ->groupBy('discussions.id')
             ->addBinding($subquery->getBindings(), 'join');
 
-        $state->setDefaultSort(function (Builder $query) use ($grammar, $value) {
-            $query->orderByRaw('MATCH('.$grammar->wrap('discussions.title').') AGAINST (?) desc', [$value]);
+        $state->setDefaultSort(function (Builder $query) use ($value, $matchTitle) {
+            $query->orderByRaw("$matchTitle desc", [$value]);
             $query->orderBy('posts_ft.score', 'desc');
         });
     }
+
+    protected function pgsql(SearchState $state, string $value): void
+    {
+        /** @var Builder $query */
+        $query = $state->getQuery();
+
+        $grammar = $query->getGrammar();
+
+        $matchCondition = "to_tsvector('english', ".$grammar->wrap('posts.content').") @@ plainto_tsquery('english', ?)";
```

Comment:
Does pgsql support language search for other languages? If so, should we make this configurable in settings?

Reply:
see comment below